ui/app/src/components/Search/utils.ts

```diff
@@ -226,8 +232,10 @@ export const useSearchState = ({ searchParameters, onSelect }: useSearchStatePro
 
   const areAllSelected = useMemo(() => {
     if (!searchResults || searchResults.items.length === 0) return false;
-    return !searchResults.items.some((item: any) => !selected.includes(item.path));
-  }, [searchResults, selected]);
+    return !searchResults.items
+      .filter((item) => !preSelectedPaths.includes(item.path))
+      .some((item: any) => !selected.includes(item.path));
```

Comment:
Are we going to 'merge' the preselectedPaths with the selected items in the search state? If so, let's say I open a browse dialog with preSelectedPaths, when on the 'onSuccess', would I have the preselected paths as part of the selection? (That would cause some issues in our current FE controls). Maybe I'm getting something wrong, I'll ping you to discuss